src/App/Fossa/Container/Analyze.hs

```diff
@@ -56,6 +56,12 @@ analyze ContainerAnalyzeConfig{..} = do
   logDebug "Running embedded syft binary"
   containerScan <- errCtx (SyftScanFailed imageLocator) (runSyft imageLocator >>= toContainerScan)
   let revision = extractRevision revisionOverride containerScan
+
+  logInfo ("Using project name: `" <> pretty (projectName revision) <> "`")
```

Comment:
I see this as one of the PR acceptance criteria, but why should we be doing this?

`Logs of finalized "project" and "revision" value for both --output and uploadScan targets`

Reply:
This is so that whichever command we run (regardless of whether the result is to be stdout'd or uploaded); we always print the project name, and revision. Previously, we were only logging project names and revisions if the result was to be uploaded. I included this in AC so, in the future defect/questions related to inferred project names and revisions are easier to diagnose.

```
fossa container analyze
fossa container analyze --output

fossa container analyze --experimental-scanner
fossa container analyze --experimental-scanner --output
```

Reply:
Sounds great to me, I love that it will be easier to get at project inference information when debugging.